<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyboard</class>
 <widget class="QMainWindow" name="keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>keyboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="keyboardgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>1261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>文本编辑</string>
    </property>
    <widget class="QLineEdit" name="keyboardeditbox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>24</y>
       <width>1011</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>***</string>
     </property>
    </widget>
    <widget class="QLabel" name="keyboardtitlelabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;xx编辑输入:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="key_language">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>en/zh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_virgule">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_dash">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_colon">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_point">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>640</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_comma">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_at">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>640</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>@</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_1">
    <property name="geometry">
     <rect>
      <x>1012</x>
      <y>560</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>560</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>560</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_4">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>480</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>480</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_6">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>480</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_7">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>400</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_8">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>400</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_9">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>400</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_0">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>640</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_q">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_w">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_e">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_r">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_t">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_y">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_u">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>u</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_i">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_o">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_p">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_s">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_d">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_f">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_g">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_h">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_j">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_k">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_l">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>480</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_z">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_c">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_v">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_b">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_n">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_m">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_caps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>caps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_ok">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>640</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_del">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>640</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="key_pinyininput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>1261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>***</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NEXTpushButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>160</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>=&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LASTpushButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>240</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_clear">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>640</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 65, 66);</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
